src/test/java/it/pagopa/pn/paperchannel/service/QueueListenerServiceImplTest.java

```diff
@@ -105,7 +107,7 @@ void nationalRegistriesResponseListenerDeliveryNotExistsTest (){
     }
 
     @Test
-    void nationalRegistriesResponseListenerReceivedDeliveryRequestOkLogTest(){
+    void nationalRegistriesResponseListenerReceivedDeliveryRequestErrormessageisnotemptyforcorrelationId(){

```

Comment:
In questo modo abbiamo sostituito un test ok con uno di KO.
Non possiamo lasciare il vecchio test rimuovendo test rimuovendo il setError e crearne uno ad-hoc per il fallimento?

Reply:
ne aggiungiamo un altro